apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx

```diff
@@ -49,102 +49,98 @@ export const ApplicationSettingsScreen = () => {
   }
 
   return (
-    <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.settings}>
-      <ScrollView bounces={false}>
-        <StatusBar type="dark" />
-
-        <SettingsSection title={strings.general}>
-          <NavigatedSettingsItem
-            icon={<Icon.Globe {...iconProps} />}
-            label={strings.selectLanguage}
-            onNavigate={() => settingsNavigation.navigate('change-language')}
-            selected={language.label}
+    <ScrollView bounces={false} style={styles.settings}>
```

Comment:
@banklesss this is a screen it's required to be wrapped around a SafeAreaView with `top` `left` `right` edges.